README.md

```diff
@@ -1,66 +1,191 @@
-# Iterator Drill: Filter
+# Iterator Drill: Map
 
 ## Learning Goals
 
-* Define how the `filter()` method works
-* Demonstrate `filter`
+* Define how the `map()` method works
+* Demonstrate `map()` with `Array`s
+* Demonstrate `map()` with complex data structures
+* Use `map()` to generate a new array
+
 
 ## Introduction
 
-We've all probably had the experience of filtering a selection of choices
-online. When shopping for clothing, we might have selected a single size so
-that we can consider only the search results we need. Or when ordering food
-online, we could have customized our results to only show restaurants currently
-open or offering delivery in our neighborhoods. Since it's such a common need
-for a user, it's also a common function for a web programmer to run.
-There are many ways to filter in programming, but the easiest is to use `filter`.
+As developers, we find ourselves responsible for all sorts of common, but
+tedious tasks, such as iterating over arrays. Although a `for` loop or nesting
```

Comment:
I think this is my error from the first round of feedback, but there should be a comma after tedious.